agent/src/agent/cli/backup.py

```diff
@@ -10,24 +10,49 @@
 
 @click.command()
 def backup():
+    """
+    Backup the Database tables and StreamSets state.
+    """
     filename = os.path.join(BACKUP_DIRECTORY, f'{AGENT_DB}_{datetime.now():%Y-%m-%d_%H:%M:%S}.dump')
     if os.system(f'pg_dump -Fc -h {AGENT_DB_HOST} -U {AGENT_DB_USER} {AGENT_DB} > {filename}') == 0:
         click.secho(f'{AGENT_DB} database successfully dumped to {filename}', fg='green')
 
 
-@click.command()
+@click.group()
+def restore():
+    """
+    Restore DataBase and StreamSets commands.
+    """
+    pass
+
+
+@restore.command()
 @click.argument('dump_file', type=click.Path(exists=True))
-def restore(dump_file):
+def database(dump_file):
+    """
+    Restore DataBase tables.
+    """
     if click.confirm(f'Are you sure you want to restore `{AGENT_DB}` database from the dump? All current data in the database will be overwritten'):
         if os.system(f'pg_restore -c -h {AGENT_DB_HOST} -U {AGENT_DB_USER} -d {AGENT_DB} {dump_file}') == 0:
             click.secho(f'Database `{AGENT_DB}` successfully restored', fg='green')
-            _restore_pipelines()
+
+
+@restore.command()
+def streamsets():
+    """
+    Restore StreamSets states with its pipelines from DataBase.
+    """
+    _restore_pipelines()
+    click.secho('StreamSets pipelines successfully restored', fg='green')
 
 
 def _restore_pipelines():
     existing, not_existing = _get_pipelines()
     for pipeline_ in not_existing:
         click.echo(f'Creating pipeline `{pipeline_.name}`')
+        if not sdc_client.get_pipeline_streamsets_stat(pipeline_):
+            click.secho(f'StreamSets `{pipeline_.streamsets.get_url()}` not available', err=True, fg='red')
+            continue
```

Comment:
let's add an option to put the pipeline to any available streamsets in case the one it was assigned to is not available